schema/paramset.json

```diff
@@ -43,7 +43,7 @@
             "$comment": "0 for 0, log_2 else"
         },
         "failure probability comment": {
-            "type": "integer"
```

Comment:
This is a change in the schema and shouldn't be in this PR.

Reply:
Agreed. I just pushed them to master directly, I assume they'll disappear from here automatically, let's see